contribs/drt/src/main/java/org/matsim/contrib/drt/passenger/events/DrtPassengerEventsReader.java

```diff
@@ -0,0 +1,79 @@
+/* *********************************************************************** *
+ * project: org.matsim.*
+ * Controler.java
+ *                                                                         *
+ * *********************************************************************** *
+ *                                                                         *
+ * copyright       : (C) 2007 by the members listed in the COPYING,        *
+ *                   LICENSE and WARRANTY file.                            *
+ * email           : info at matsim dot org                                *
+ *                                                                         *
+ * *********************************************************************** *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *   See also COPYING, LICENSE and WARRANTY file                           *
+ *                                                                         *
+ * *********************************************************************** */
+
+package org.matsim.contrib.drt.passenger.events;
+
+import org.matsim.api.core.v01.Id;
+import org.matsim.api.core.v01.network.Link;
+import org.matsim.api.core.v01.population.Person;
+import org.matsim.contrib.dvrp.optimizer.Request;
+import org.matsim.contrib.dvrp.passenger.DvrpPassengerEventsReader;
+import org.matsim.core.api.experimental.events.EventsManager;
+import org.matsim.core.events.MatsimEventsReader;
+import org.matsim.core.utils.io.MatsimXmlParser;
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+
+import java.util.Map;
+import java.util.Stack;
+
+public final class DrtPassengerEventsReader extends MatsimXmlParser {
+
+    private DvrpPassengerEventsReader delegate;
+
+    public DrtPassengerEventsReader(EventsManager events) {
+        delegate = new DvrpPassengerEventsReader(events);
+        this.setValidating(false);
+        delegate.addCustomEventMapper(org.matsim.contrib.drt.passenger.events.DrtRequestSubmittedEvent.EVENT_TYPE, getDrtRequestSubmittedEventMapper());
+    }
+
+    private MatsimEventsReader.CustomEventMapper<org.matsim.contrib.drt.passenger.events.DrtRequestSubmittedEvent> getDrtRequestSubmittedEventMapper() {
```

Comment:
There are many looooong class names (prefixed with packages) in this class. They should go to imports.